handler/commentHandler.go

```diff
@@ -506,3 +515,41 @@ func getMultiLabelLimit() int {
 	}
 	return labelLimitDefault
 }
+
+func createMessage(req types.IssueCommentOuter, cmdType, cmdValue string, config config.Config, derekConfig *types.DerekRepoConfig) (string, error) {
+	var err error
+
+	var buffer bytes.Buffer
+
+	buffer.WriteString(fmt.Sprintf("%s wants to add message of type '%s' on issue #%d \n", req.Comment.User.Login, cmdValue, req.Issue.Number))
+
+	messageValue, err := createIssueComment(derekConfig.Messages, cmdValue)
+	if err != nil {
+		return buffer.String(), fmt.Errorf("Error while filtering message: %s", err.Error())
+	}
+
+	buffer.WriteString(fmt.Sprintf("Message '%s' with content: \n%s\nfound.\n", cmdValue, *messageValue.Body))
+
+	client, ctx := makeClient(req.Installation.ID, config)
+
+	comment, resp, err := client.Issues.CreateComment(ctx, req.Repository.Owner.Login, req.Repository.Name, req.Issue.Number, messageValue)
+	if err != nil {
+		return buffer.String(), err
+	}
+	buffer.WriteString(fmt.Sprintf("Successfully applied comment: \n%s\nstatus code: %d",
+		*comment.Body,
+		resp.StatusCode))
+
+	return buffer.String(), nil
+}
+
+func createIssueComment(messages []types.Message, wantedMessage string) (*github.IssueComment, error) {
+	for _, message := range messages {
+		if message.Name == wantedMessage {
+			return &github.IssueComment{
+				Body: &message.Value,
+			}, nil
+		}
+	}
+	return nil, fmt.Errorf("Unable to find message: %s", wantedMessage)
```

Comment:
Can we change the error to explain that no such message has been configured?

Reply:
Yep, error is now `"Message: docs is not configured` `docs` is example here